R/plumb-globals.R

```diff
@@ -48,8 +48,8 @@ plumbOneGlobal <- function(fields, argument, envir = parent.frame()){
            fields$info$version <- def
          },
          apiTag={
-           tagMat <- stri_match(def, regex="^\\s*(\\w+)\\s+(\\S.+)\\s*$")
-           name <- tagMat[1,2]
+           tagMat <- stri_match(def, regex="^\\s*(\"[^\"]+\"|'[^']+'|\\w+)\\s+(\\S.+)\\s*$")
+           name <- stri_trim_both(tagMat[1,2], pattern = "[[\\P{Wspace}]-[\"']]")
```

Comment:
Can we make this a function so `plumb-block.R` can also use it?

```suggestion
           tagMat <- stri_match(def, regex="^\\s*(\"[^\"]+\"|'[^']+'|\\S+)\\s+(\\S.+)\\s*$")
           name <- stri_trim_both(tagMat[1,2], pattern = "[[\\P{Wspace}]-[\"']]")
```

Reply:
I do not think so.
The block tag regex does not accept parameters while the global api one does.
Let's keep it like this for now. I will revisit this when I gather the courage to refactor plumb-globals and plumb-block.